tool/pmt-builder.js

```diff
@@ -0,0 +1,21 @@
+const mempoolJS = require("@mempool/mempool.js");
+const pmtBuilder = require("../index");
+
+(async () => {
+    try {
+        const { bitcoin: { blocks } } = mempoolJS({
+            hostname: 'mempool.space',
+            network: 'testnet' // 'testnet' | 'mainnet'
+        });
+    
+        const hash = '00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e';
+        const blockTxids = await blocks.getBlockTxids({ hash });
+    
+        const resultPmt = pmtBuilder.buildPMT(blockTxids, blockTxids[0]);
+        console.log("Block Transactions Ids:", blockTxids);
+        console.log("Filtered Hash:", blockTxids[0]);
+        console.log("Result:", resultPmt);
+    } catch (e) {
+        console.log(e);
+    }
+})();
```

Comment:
It would be nice to receive the network, block hash, and filtered tx hash via console param.

To use it like this
`node tool/pmt-builder.js network blockHash txHash`

We should also update the README, adding that this tool exists and how to use it

Reply:
Done